index.html

```diff
@@ -8,43 +8,114 @@
-                   <div class="baby-flex"><i class="fas fa-share"></i><div>share</div></div>
+    <!-- main section -->
+    <main class="main-container">
+        <div class="mediaPlayer"><div id="main-frame"><iframe src="https://www.youtube.com/embed/V74l_zS1x8E" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
+            <h3 class="title">Contributing to Open Source Part 1: The Essay Way</h3>
+               <div class="favicons-row">
+                   <span>57,800 Views * jun 9, 2020</span>
+                   <div id="thumbs-up"><a href="#"><i class="fas fa-thumbs-up"></i><span class="favicons-text" >425</span></a></div>
+                   <div id="thumbs-down"><a href="#"><i class="fas fa-thumbs-down"></i><span class="favicons-text">18</span></a></div>
+                   <div id="share"><a href="#"><i class="fas fa-share"></i><span class="favicons-text">Share</span></a></div>
+                   <div id="save"><a href="#"><img src="image/menu-plus.jpg" alt="dif-menu-icon"><span id="save-btn">Save</span></a></div>
+                   <div id="ellipse"><a href="#"><i class="fas fa-ellipsis-h"></i></a></div>
                </div>
+            
+               <div class="videoInfo">
+                <div><a href="#"><img id="odin-logo" src="image/odin.png" alt="odin project logo">
+                        <div class="name-views">
+                            <div><h4 id="vid-name">The Odin Project</h4></div>
+                            <div><span>993 Subscribers</span></div>
+                        </div>
+                    </a>
+                </div>
+                <div>
+                    <button id="subscribe" type="button"><span id="button-text">SUBSCRIBE</span></button>
+                    <span id="bell-notifier" ><a href=""><img  src="image/scribe-bell.jpg" alt="subscribe-btn"></a></span>   
+                </div>
+              </div>
+              <div><p class="main-p">How do you contribute to open source? in this video, I'll describe the easiest way to make a contribution to open source projects...<a href="">More</a> </p></div>
         </div>
+        <!-- Aside section -->
         <div class="listedView">
-            <div class="iconcue"><img src="image/firstfoto.jpg" alt=""></div> 
-            <div><img src="image/img5.jpg" alt="" class="iconcue"></div>
-            <div><img src="image/img4.jpg" alt="" class="iconcue"></div>
+            <div class="privcons">
+                <div id="upNext" ><a href="#">Up Next</a></div>
+                <div id="autoP" ><a href="#">AUTOPLAY</a></div>
+                <div id="toggle-btn" ><a href="#"><i class="fas fa-toggle-on"></i></a></div>
+            </div>
+            <div class="listed-video-style">
+                <div><iframe src="https://www.youtube.com/embed/YJZCUhxNCv8" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
```

Comment:
Video previews shouldn't be displayed, putting some placeholder images will do for this project.